app/templates/auth/callback.hbs

```diff
@@ -0,0 +1,2 @@
+{{page-title "Callback"}}
```

Comment:
I think that the auth templates are not needed (callback, login, logout, switch) because we don't provide pages in our applications for it, it all happens in acmdim's world